### DISCUSSION TITLE
NextJS URL multiple parameters passing

### DISCUSSION THREAD

**Discussion by saresh-krish** [vercel/next.js#14314] [Help]
Hi
I am using import { useRouter } from "next/router"; 
### Currently what I am getting is.
http://localhost:3000/app/file?id=abc35a5e-1234-9644-acb3-0242ac760002&subject='1234'
### What I need
http://localhost:3000/app/file/abc35a5e-1234-9644-acb3-0242ac760002/1234

I do not want parameters name on Url .

### Routing Link 
`<Link href={`/app/filel?id=${id}&subject=${"1234"}`} >`

### Server File:
server.get('/app/file/:id/:subject', (req, res) => {
        return app.render(req, res, '/app/file', { id: req.params.id, subject: req.params.subject });
    });

**### System information**
OS: Linux
Browser : Chrome
"next": "^9.3.4",
"react": "^16.13.1",

**Comment by jamesmosier** [comment]
I think you'd want to do something like...

```js
server.get('/app/file/:id/:subject', (req, res) => {
  return app.render(req, res, '/app/file', { ...req.params, ...req.query });
});
```
